### DISCUSSION TITLE
ROSIE access

### DISCUSSION THREAD

**Discussion by Deena-lab** [RosettaCommons/rosetta#93] [ROSIE] [answered]
I'd like to access ROSIE but I'm getting an error that "Your GitHub account does not have verified academic email address! Please visit https://github.com/settings/emails and setup it! If your institution _is academic_ please contact our support forum at [https://github.com/RosettaCommons/rosetta/discussions/categories/rosie] and provide your institution name and domain address."
 I currently work at Okinawa Christian School International (www.ocsi.org). Would you please give me access.

**Comment by lyskov** [comment, marked as answer]
Thank you for letting us know! - this should be fixed now. Please feel free to re-open if you still having trouble logging in.

**Reply by Deena-lab** [reply]
Thanks so much!
